### ISSUE TITLE
RSS for non-political Government agencies

### ISSUE THREAD

**Issue by vietnguyengit** [gov-rss/rss#1]
Hi mate,

Just wonder if media releases from non-political Federal and State Gov agencies are in the scope of your project?

And I think I know you in person some years ago 🤔

Cheers

**Comment by callumskeet** [comment]
Yeah, I think we worked together for a bit back in my cafe days. Hope you're keeping it :100:

Federal's definitely in scope, which agency did you have in mind?

**Comment by vietnguyengit** [comment]
Cheers mate, yeah I relocated to Tassie and have been working for a Federal government-funded research orgnisation called Integrated Marine Observing System. I remember that you had a job at BOM a few years ago. Are you still working for them?

The list below is agencies that I'm personally interested in:

- https://www.homeaffairs.gov.au/
- https://www.finance.gov.au/
- https://treasury.gov.au/
- https://www.industry.gov.au/
- https://www.csiro.au/en/
- https://www.ga.gov.au/
- https://www.servicesaustralia.gov.au/
- https://www.aims.gov.au/
- https://www.abs.gov.au/
- https://parksaustralia.gov.au/
- https://www.dcceew.gov.au/
- http://www.bom.gov.au/
 
I can help contribute PRs to your project, let me know if you want some extra help. Cheers.

**Comment by callumskeet** [comment]
I'm working on another side project at the moment. So I'm not sure when I'll get around to working doing any of these. 

I'll happily accept any PRs you raise though.

Also double check if any of these agencies already have an RSS feed. I know BOM has one for their media releases: https://media.bom.gov.au/rss/ (also for warnings and climate info http://www.bom.gov.au/rss/)
